x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -9,7 +9,7 @@ pub type Gas = u64;
 
 /// A struct that enforces a fixed length of 32 bytes which represents an address.
 #[cfg_attr(feature = "debug", derive(Debug))]
-#[derive(Clone, Copy, PartialEq, Eq, BorshSerialize, BorshDeserialize, Hash)]
+#[derive(Clone, Copy, PartialEq, Eq, BorshSerialize, BorshDeserialize, Hash, PartialOrd, Ord)]
```

Comment:
This is to be able to write an `Address` in an `HashSet` just like it is for the `voters` field.

Reply:
Oh right, thanks. This is to be able to have the `voters` field in the `Proposer` struct which implements `BorshSerialize`, It's a requirement because maps are sorted by key when serialized.
I think that I will just store the `voters` in a separate state key.